cypress/integration/cucumber/step_definitions/MFE/login.test.js

```diff
@@ -0,0 +1,37 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+
+/// <reference types="Cypress" />
+
+beforeEach(() => {
+  cy.initiateExceptionsLogger();
+});
+
+afterEach(() => {
+  cy.end();
+});
+
+Given('User opens the websites login page', () => {
+  cy.visit('/');
+});
+
+When('User enters username {string}', (username) => {
+  cy.get('#username').type(username);
+});
+
+And('User clicks continue', () => cy.get('.iw0iHcFjY--gDZi9jUQ1_').click());
```

Comment:
Why not add `id`'s, what if with versions this `className` changes ? ( If, it happens to be randomly generated )

Reply:
Just curious

Reply:
@harshit151001 It is not React/webpack which causes these classnames, actually  SCSS (or .sass) files  get compiled into a regular a CSS file and the newly compiled CSS file is what gets loaded to the browser to style the web applications. So I guess it should be safe to use until this mechanism changes.

Reply:
I wouldn't use generated classnames except as a last resort. Why not just ise `cy.get('button')`, given that there's only a single button on the screen? Alternatively, take advantage of the fact that this repo has the [Cypress Testing Library](https://testing-library.com/docs/cypress-testing-library/intro/) and use `cy.findByText('Continue')`

Reply:
dear @ibacher i would still not use generated attribute values since they are dynamic per build, also using   `cy.get('button')` is assurance for a future maintanance burden.

I would perhaps consider it a design recommendation to have more reliable element trackers.
`cy.findByText('Continue')` looks more convincing